src/rules/newline-after-import.js

```diff
@@ -87,7 +88,7 @@ module.exports = {
         nextNode = nextNode.decorators[0];
       }
 
-      const options = context.options[0] || { count: 1 };
+      const options = context.options[0] && 'count' in context.options[0]? context.options[0] : { count: 1 };
```

Comment:
```suggestion
      const options = context.options[0] && 'count' in context.options[0] ? context.options[0] : { count: 1 };
```
what happens when `count` is not provided but `considerComments` is?